jupyter-book/predictive_modeling_pipeline/03_categorical_pipeline_video.md

```diff
@@ -0,0 +1,3 @@
+# 🎥 How to define a scikit-learn pipeline and visualize it
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5ri7BhFrNe4" title="YouTube video player" frameborder="0" rel="0" showinfo="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Adding `?rel=0` to the `src` should make is possible to avoid the suggested videos from other channels.

```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/5ri7BhFrNe4?rel=0" title="YouTube video player" frameborder="0" rel="0" showinfo="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Reply:
There is a problem on the layout:

https://2461-246063957-gh.circle-artifacts.com/0/jupyter-book/predictive_modeling_pipeline/03_categorical_pipeline_video.html

It's using all the vertical space which I find really annoying. I am not yet sure why.

Reply:
It's from `sklearn_mooc.css`:

```css
iframe {
    /* The constant in these values must be adjusted if the theme changes */
    /*! height: calc(100vh - 14ex); */
    /*! position: fixed; */
    /*! z-index: 1000; */
    /*! border-color: #AAA; */
    /*! border-style: solid; */
}
```

@lesteve any idea why this is needed in the first place? Is this for the slides? If so we can probably specialize this class to apply only to the slides.

Reply:
I think Gaël did this, I think this was for slides on small screens.